src/api/Infrastructure/Shrooms.Infrastructure/ExcelGenerator/ExcelBuilder.cs

```diff
@@ -14,12 +15,24 @@ public ExcelBuilder()
         }
 
         public IExcelBuilder AddNewWorksheet(string sheetName, IEnumerable<string> headerItems, IEnumerable<IEnumerable<object>> rows)
+        {
+            var worksheet = _package.Workbook.Worksheets.Add(sheetName);
+            var worksheetBuilder = new ExcelWorksheetBuilder(worksheet);
+
+                worksheetBuilder
+                    .WithHeader(headerItems)
+                    .WithRows(rows)
+                    .Build();
+
+            return this;
+        }
+
+        public IExcelBuilder AddNewWorksheet(string sheetName, IEnumerable<IEnumerable<object>> rows)
         {
```

Comment:
Using method overloading I added an option to create worksheet without needing the header.